packages/shared/src/components/widgets/ReferralSocialShareButtons.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactElement, useContext } from 'react';
+import { Button } from '../buttons/Button';
+import {
+  getFacebookShareLink,
+  getTwitterShareLink,
+  getWhatsappShareLink,
+  ShareProvider,
+} from '../../lib/share';
+import WhatsappIcon from '../icons/Whatsapp';
+import FacebookIcon from '../icons/Facebook';
+import TwitterIcon from '../icons/Twitter';
+import { link } from '../../lib/links';
+import { labels } from '../../lib';
+import { SimpleTooltip } from '../tooltips';
+import { AnalyticsEvent, TargetType } from '../../lib/analytics';
+import AnalyticsContext from '../../contexts/AnalyticsContext';
+
+const ReferralSocialShareButtons = ({
+  url,
+  origin,
+}: {
+  url: string;
+  origin: TargetType;
```

Comment:
Origin is a different property we send on certain events. Let's use `targetType` to avoid confusion. Also maybe we should introduce `ReferralSocialShareButtonsProps` to clean it a bit.

```
interface ReferralSocialShareButtonsProps {
  url: string;
  origin: TargetType;
}
```

```suggestion
  targetType: TargetType;
```

Reply:
Agree with Lee in this case.
Did open a [discussion](https://dailydotdev.slack.com/archives/C02JAUF8HJL/p1699946558816669) for experiment team to give us better clarity, seems like they randomly assign the properties now.

Reply:
On it

Reply:
Done, thank u!